addons/point_of_sale/tools/posbox/overwrite_before_init/etc/init.d/odoo

```diff
@@ -20,6 +20,7 @@ DESC=odoo
 CONFIG=/home/pi/odoo/addons/point_of_sale/tools/posbox/configuration/odoo.conf
 LOGFILE=/var/log/odoo/odoo-server.log
 PIDFILE=/var/run/${NAME}.pid
+MODULES=$(cd /home/pi/odoo/addons/ && ls -m | tr -d ' ')
```

Comment:
With this command you list directories and files.
And you can give a specific directory to list without `cd`
Maybe a solution : `ls $HOME/odoo/addons/ -dm */ | tr -d ' /'`

Reply:
There should be only directories in `home/pi/odoo/addons` so I didn't put `-d`, but I can add it.

For the second comment, the problem is if I use `ls $HOME/odoo/addons/ -dm */`, it will list directories as `/home/pi/odoo/addons/hw_drivers,/home/pi/odoo/addons/web,...`. So I need to use `cd` to retrieve only the name of the directory.